src/test/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupportSpec.scala

```diff
@@ -17,6 +18,117 @@ class DataUseRestrictionSupportSpec extends FreeSpec with Matchers with DataUseR
 
     "Structured Use Restriction" - {
 
+      "when questionnaire answers are used to populate restriction fields" - {
+
+        "and all consent codes are true or filled in" in {
+          val ontologyDAO = new MockOntologyDAO
+          val request = StructuredDataRequest(generalResearchUse = true,
+            healthMedicalBiomedicalUseOnly = true,
+            diseaseUseOnly = Array(4325,2531),
+            commercialUseProhibited = true ,
+            forProfitUseProhibited = true,
+            methodsResearchProhibited = true,
+            aggregateLevelDataProhibited = true,
+            controlsUseProhibited = true,
+            genderUseOnly = "female",
+            pediatricResearchOnly = true,
+            IRB = true,
+            prefix = "blah")
+
+          val expected = Map("blahconsentCodes" -> Array("NPU","RS-G","NCU","HMB","RS-FM","NCTRL","RS-PD","IRB","NAGR","GRU","NMDS","DS:Ebola hemorrhagic fever","DS:hematologic cancer").toJson,
```

Comment:
The example in the API Doc indicated that there wasn't an in-built delimiter. (the example prefix was "library:" in the request) I assume this was so that users could choose their own delimiters.